app/assets/stylesheets/application.scss

```diff
@@ -74,3 +103,18 @@ ul {
   margin-right: 5px;
   width: 50px;
 }
+
+div .search-bar-only {
+  background-color: rgba(60, 59, 59, 0.3);
+  border-color: $white;
+  -moz-border-radius: 15px;
+  -webkit-border-radius: 15px;
+  border-radius: 15px;
+  color: $white;
+  width: 200px;
+}
+
+body {
+  background-image: url("http://pcwallart.com/images/white-snowflake-background-wallpaper-2.jpg");
```

Comment:
URL `http://pcwallart.com/images/white-snowflake-background-wallpaper-2.jpg` should not contain protocol or domain